### ISSUE TITLE
Tabs Keyboard Shortcuts

### ISSUE THREAD

**Issue by isontheline** [isontheline/pro.webssh.net#627]
Maybe add keyboard to allow for faster switching between terminal and SFTP tabs? And could be compatible with iPad (what I'm using) so could use option or command key (would then also be compatible with MacOS)? Could be similar to how safari deals with switching tabs (command + number to switch to that number tab, e.g. cmd + 1 switches to leftmost tab).

_Originally posted by @Peoppon in https://github.com/isontheline/pro.webssh.net/discussions/626_

**Comment by AInteriorB** [comment]
Great idea! On MacOS switching between tabs is done with control+TAB or control+shift-TAB. And switching between windows is done with cmd+>. The only cave is that these combinations are used inside the console.

**Comment by isontheline** [comment]
You're right @AInteriorB the control key alone shouldn't be used in order to don't hurt the console
